packages/ng/container/container.component.ts

```diff
@@ -0,0 +1,26 @@
+import { booleanAttribute, Component, input, ViewEncapsulation } from '@angular/core';
+
+@Component({
+	selector: 'lu-container',
+	standalone: true,
+	styleUrl: './container.component.scss',
+	template: '<ng-content />',
+	encapsulation: ViewEncapsulation.None,
+	host: {
+		class: 'container',
+		'[class]': 'classesConfig',
+	},
+})
+export class ContainerComponent {
+	center = input(false, { transform: booleanAttribute });
+	overflow = input(false, { transform: booleanAttribute });
+	max = input<null | 'M' | 'L' | 'XL' | 'XXL' | 'XXXL'>(null);
```

Comment:
Is adding `null` necessary?

```suggestion
	max = input<null | 'M' | 'L' | 'XL' | 'XXL' | 'XXXL'>();
```